README.md

```diff
@@ -1,3 +1,7 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+# Creating Environments variables
+Two .env files must be created. one for the test database and the other for the development database. 
+The test .env file should connect to the nc_news_test database
```

Comment:
Personally I would be more explicit with the naming of the .env file. If someone were not familiar with using dotenv, this could be a little confusing to read initially.

.env.test - test 
.env.development - development